### ISSUE TITLE
Definitions

### ISSUE THREAD

**Issue by XavierGuertin** [XavierGuertin/CondoVision#3]
Type:

feature: New features or requests.
bug: Something isn't working.
enhancement: Improvements or additions to existing features.
documentation: Related to documentation (README, wiki, etc.).
refactor: Code changes that neither fix a bug nor add a feature.
test: Adding or updating tests.

Priority:

critical: Requires immediate attention.
high: Important issue that should be resolved soon.
medium: Regular priority.
low: Low priority issues.
Status:

help wanted: Extra attention is needed.
good first issue: Good for newcomers.
in progress: Work is currently being done.
blocked: Cannot proceed until another issue is resolved.
review needed: Needs review before proceeding.
on hold: Not a current focus, to be addressed later.

Scope:

frontend: Issues related to the frontend.
backend: Issues related to the backend.
database: Database related issues.
api: Issues related to API.
security: Security-related issues.

Effort/Complexity (if you're using story points, you may adapt these):

1 pt: Trivial, very little effort.
2 pts: Small effort.
3 pts: Medium effort.
5 pts: Large effort.
8+ pts: Very large/complex task.
